src/optimizer/cmaes_optimizer/evalCMAESOutput.py

```diff
@@ -0,0 +1,38 @@
+#!/bin/python
```

Comment:
How is evalCMAESOutput.py different from evalPSOOutput.py?

Reply:
CMAES and PSO output files have different formats. Each script matches the corresponding format.